fe/fe-common/src/main/java/org/apache/doris/common/io/DiskUtils.java

```diff
@@ -0,0 +1,122 @@
+        public long available;
+        public int useRate;
+        public String mountedOn = "";
+    }
+
+    public static Df df(String dir) {
+        String os = System.getProperty("os.name");
+        if (os.startsWith("Windows")) {
+            Df df = new Df();
+            df.available = Long.MAX_VALUE / 1024;
+            return df;
+        }
+
+        Process process;
+        try {
+            process = Runtime.getRuntime().exec("df " + dir);
+            InputStream inputStream = process.getInputStream();
+            BufferedReader reader = new BufferedReader(new InputStreamReader(inputStream));
+
+            // Filesystem      1K-blocks       Used Available Use% Mounted on
+            // /dev/sdc       5814186096 5814169712         0 100% /home/spy-sd/sdc
+            String titleLine = reader.readLine();
+            String dataLine = reader.readLine();
+            if (titleLine == null || dataLine == null) {
+                return null;
+            }
+            String[] values = dataLine.split("\\s+");
+            if (values.length != 6) {
+                return null;
+            }
+
+            Df df = new Df();
+            df.fileSystem = values[0];
+            df.blocks = Long.parseLong(values[1]);
+            df.used = Long.parseLong(values[2]);
+            df.available = Long.parseLong(values[3]);
+            df.useRate = Integer.parseInt(values[4].replace("%", ""));
+            df.mountedOn = values[5];
+            return df;
+        } catch (IOException e) {
+            e.printStackTrace();
+            return null;
+        }
+    }
+
+    private static List<String> getTitles(String titlesLine) {
+        List<String> titles = new ArrayList<>();
+        String[] titleArray = titlesLine.split("\\s+");
+        for (String title : titleArray) {
+            if (title.equalsIgnoreCase("on")) {
+                if (!titles.isEmpty()) {
+                    int lastIdx = titles.size() - 1;
+                    titles.set(lastIdx, titles.get(lastIdx) + "On");
+                }
+            } else {
+                titles.add(title);
+            }
+        }
+        return titles;
+    }
+
+    private static String[] units = new String[]{"", "K", "M", "G", "T", "P"};
+
+    public static String sizeFormat(long size) {
+        int unitPos = 0;
+        while (size > 1024 && unitPos < units.length - 1) {
+            unitPos++;
+            size /= 1024;
+        }
+        return size + units[unitPos];
+    }
+
+    public static void main(String[] args) {
+        long [] values = new long[]{
+                1L,
+                1L * 1024,
+                1L * 1024 * 1024,
+                1L * 1024 * 1024 * 1024,
+                1L * 1024 * 1024 * 1024 * 1024,
+                1L * 1024 * 1024 * 1024 * 1024 * 1024,
+                1L * 1024 * 1024 * 1024 * 1024 * 1024 * 1024,
+        };
+
+        for (long v : values) {
+            System.out.println(sizeFormat(v));
```

Comment:
Is this for debugging? Please add unit tests or regression tests

Reply:
Add a unit test class.